scripts/globals/mobskills/entice.lua

```diff
@@ -11,9 +12,19 @@ function onMobSkillCheck(target,mob,skill)
 end
 
 function onMobWeaponSkill(target, mob, skill)
-    local typeEffect = tpz.effect.SLEEP_II
+    local typeEffect = tpz.effect.CHARM_I
+    local power = 0
 
-    skill:setMsg(MobStatusEffectMove(mob, target, typeEffect, 1, 0, 20))
+    if (not target:isPC()) then
```

Comment:
just a heads up: outer parens aren't needed
```lua
if not target isPC() then
```
works

Reply:
This is what I get for aggressive copy/paste...

Reply:
:spaghetti: